components/text/dwdiff/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Matthew R. Trower
+#
+
+BUILD_BITS=64
```

Comment:
This is default, so please remove it.

Reply:
Alright.  It is part of templates/configure.mk that I used as a base, so maybe it should be removed from that template file?  Or are those templates not meant to be used as basis for new components.

Reply:
Yes, that (the second thought).  Templates will never cover all possible cases.